defaultPnorLayoutSingleSide.xml

```diff
@@ -174,6 +174,8 @@ Layout Description
         <physicalOffset>0x961000</physicalOffset>
         <physicalRegionSize>0x100000</physicalRegionSize>
         <side>A</side>
+        <xz/>
+        <xzSize>0x100000</xzSize>
```

Comment:
That's what I was thinking when I made this change as well. Our current region sizes aren't exact, so it's not really that important for the compressed size to be exact either.  Also, I kept the same size because I didn't want to put down an arbitrary number that was probably going to be changed in the future anyway, and since the skiboot image isn't allowed to be bigger by default, it doesn't really matter.  2x to 8x is probably a good guess, so we could change it if you think that's a good idea. And xz-embedded looks for a footer in the image so if we pass in something smaller than the image size, it won't matter.